src/app/core/services/issue.service.ts

```diff
@@ -35,6 +35,7 @@ export class IssueService {
   private issueTeamFilter = 'All Teams';
   readonly MINIMUM_MATCHES = 1;
   readonly userRole = UserRole;
+  private isModified = false;
```

Comment:
What is the reason for introducing this Boolean variable?

Reply:
This boolean is for the "Sync" function

Reply:
I suppose the usage of this Boolean attribute will change in the future when we do polling? If so, can add a comment to document this?

Reply:
You can define your function to have a default value, thereby making the last parameter optional.

``` typescript
getIssueComments(issueId: number, forceReload = false) {
  ...
}
```
With this, we can remove the `isModified` flag.

Reply:
@JunWei96 Sry I'm not sure how to set the forceReload, the isModified is set in another function `reloadAllIssues()`

Reply:
Yes, `isModified` flag is ultimately used in `createIssueModel` right?

Can we do something like:

```typescript
createIssueModel(issueInJson: {}, forceReloadComments = false)
{
  ...
} 
```

Reply:
@JunWei96 I might have to change `initializeData()` as well, is there a reason for using optional parameter instead of using the `isModified` flag?

Reply:
Yeap sure, lets rename it then.